particle/commands.md

```diff
@@ -1,77 +1,142 @@
 # Commands
 
-Start the local webpack-dev-server for Pattern Lab, hot-reload all frontend assets, automatically recompile all assets and Pattern Lab on file change.
+## Start a Server
+
+Install dependencies
+
+```bash
+npm install
+```
+
+Setup and/or reinstall Pattern Lab
+
+```bash
+npm run setup
+```
+
+Start the local Webpack dev server for Pattern Lab, hot-reload all frontend assets, and automatically recompile all assets and Pattern Lab on file change
 
 ```bash
 npm start
 ```
 
-Start file watchers and compile assets to disk for Drupal on changes.
+Start file watchers and compile assets to disk for Drupal on changes
 
 ```bash
 npm run dev:drupal
 ```
 
-Compile production assets for Pattern Lab \(e.g. for a static file host\):
+Start file watchers and compile assets to disk for Grav on changes
 
 ```bash
-npm run build:pl
+npm run dev:grav
 ```
 
+## Compile Assets
+
 Compile production assets for Drupal
 
 ```bash
 npm run build:drupal
 ```
 
-Reinstall and setup Pattern Lab
+Compile production assets for Grav
 
 ```bash
-npm run setup
+npm run build:grav
 ```
 
+Compile production assets for Pattern Lab \(e.g. for a static file host\)
+
+```bash
+npm run build:pl
+```
+
+## Code Formatting
+
+Lint and Prettier both JavaScript and Sass
+
+```bash
+npm run fmt
+```
+
+Lint and Prettier only JavaScript
+
+```bash
+npm run fmt:js
+```
+
+Lint and Prettier only Sass
+
+```bash
+npm run fmt:scss
+```
+
+## Code Linting
+
 Run all linters:
 
 ```bash
 npm run lint
 ```
 
-Run only Javascript linters:
+Run only Javascript linters
 
 ```bash
 npm run lint:js
 ```
 
-Run only Sass linters:
+Run only Sass linters
 
 ```bash
 npm run lint:scss
 ```
 
-Run all tests:
+Run only Twig linters
 
 ```bash
-npm test
+npm run lint:twig
 ```
 
-Run only unit test:
+Run only Twig linters for Drupal
 
 ```bash
-npm run test:unit
+npm run lint:twig:drupal
 ```
 
-Run only pa11y accessibility tests:
+Run only Twig linters for Grav
 
 ```bash
-npm run test:pa11y
+npm run lint:twig:grav
 ```
 
-Run Yeoman generator to make new component:
+Run only Twig linters for Pattern Lab
 
 ```bash
-npm run new
+npm run lint:twig:source
+```
+
+## Automated Testing
+
+Run all tests:
+
+```bash
+npm test
+```
+
+Run only Pa11y accessibility tests
```

Comment:
Pa11y isn't a dev dependency so needs to be installed by user before we can run this command. Not sure if we want to add that here.